us/en/skymiles/blocks/feedback/feedback.js

```diff
@@ -0,0 +1,83 @@
+function showAcknowledgeText() {
+  // Replace question with acknowledge text
+  const acknowledgeText = document.getElementById('acknowledge-text');
+  const questionContainer = document.querySelector('.question-container');
+  questionContainer.setAttribute('hidden', 'true');
+  acknowledgeText.removeAttribute('hidden');
+}
+
+export default async function decorate(block) {
+  block.innerHTML = '';
+  const divSectionFeedback = document.createElement('div');
+
+  const spanAckText = document.createElement('span');
+  spanAckText.setAttribute('id', 'acknowledge-text');
+  spanAckText.textContent = 'Thank you for your feedback.';
+  spanAckText.setAttribute('hidden', 'true');
+
+  const inputSurveyID = document.createElement('input');
+  inputSurveyID.setAttribute('type', 'hidden');
+  inputSurveyID.setAttribute('id', 'surveyID');
+  inputSurveyID.setAttribute('name', 'surveyID');
+  inputSurveyID.setAttribute('value', 'Y0LLcnRhFy9KjxsjyPybemlqYF1spmda');
+
+  const divQuestionContainer = document.createElement('div');
+  divQuestionContainer.classList.add('question-container');
+
+  const spanAckQuestion = document.createElement('span');
+  spanAckQuestion.classList.add('acknowledge-question');
+  spanAckQuestion.textContent = 'Did you find the information you were looking for on this page?';
```

Comment:
Is this always the same message? I wonder if we should let the author potentially configure the question in the block content.

Reply:
@ramboz I thought about this. Yeah, may be we could take the approach of the `TOC` block. There would be a default message, which they could override. What do you think?

Reply:
Sounds like a great idea!